taxi_service/urls.py

```diff
@@ -22,4 +22,5 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", include("taxi.urls", namespace="taxi")),
+    path("accounts/", include("django.contrib.auth.urls")),
 ] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
Serving static files using `django.conf.urls.static.static` is suitable for development but not recommended for production. In a production environment, it's better to serve static files using a web server like Nginx or Apache.